tests/api_fastapi/core_api/routes/public/test_assets.py

```diff
@@ -459,3 +460,32 @@ def test_should_respond_404(self, test_client):
         )
         assert response.status_code == 404
         assert response.json()["detail"] == "The Asset with uri: `s3://bucket/key` was not found"
+
+
+class TestPostAssetEvents(TestAssets):
+    def test_should_respond_200(self, test_client, session):
+        self.create_assets()
+        event_payload = {"asset_uri": "s3://bucket/key/1", "extra": {"foo": "bar"}}
+        response = test_client.post("/public/assets/events", json=event_payload)
+
+        assert response.status_code == 200
+
+        assert response.json() == {
+            "id": mock.ANY,
+            "asset_id": mock.ANY,
+            "uri": "s3://bucket/key/1",
+            "extra": {"foo": "bar", "from_rest_api": True},
+            "source_task_id": mock.ANY,
+            "source_dag_id": mock.ANY,
+            "source_run_id": mock.ANY,
+            "source_map_index": -1,
+            "created_dagruns": [],
+            "timestamp": mock.ANY,
+        }
```

Comment:
We are missing the `_check_last_log` compared to the legacy test. I think it is a nice addition to keep.

Reply:
@pierrejeambrun I tried adding `_check_last_log` test and noticed it failed. The reason is there is no entry going in log table to event as `api.create_asset_event` as mentioned [here](https://github.com/astronomer/airflow/blob/f671fcef549e638d1551c4ad98cea2a2d2794824/tests/api_connexion/endpoints/test_asset_endpoint.py#L606) and [this](https://github.com/astronomer/airflow/blob/fdde5acd57b67f7654696dd32291472a34522a47/tests_common/test_utils/www.py#L89) assert was failing.

Reply:
Yes, indeed we cannot `_check_last_log` yet in the FastAPI API. I created an issue for that, we need first to do:
https://github.com/apache/airflow/issues/44057

Marking as resolved, you can ignore my comment.